src/components/result-pager/result-pager.tsx

```diff
@@ -21,7 +32,7 @@ function ResultPager({
       </p>
       <button
         type="button"
-        className="btn-primary btn-outline btn-medium arrow__hover--right-small"
+        className="test-result-pager btn-primary btn-outline btn-medium arrow__hover--right-small"
```

Comment:
Test?

Reply:
So Imma change this in another pr, and just... fix it everywhere (https://docs.cypress.io/guides/references/best-practices)